1.5/Source/PawnStorages/PawnStoragesModule-VEF/HarmonyPatches/CompPawnStorageNutrition_Patches.cs

```diff
@@ -0,0 +1,70 @@
+﻿using HarmonyLib;
+using PipeSystem;
+using RimWorld;
+using UnityEngine;
+
+namespace PawnStorages.VEF.HarmonyPatches;
+
+[HarmonyPatch(typeof(CompPawnStorageNutrition))]
+public static class CompPawnStorageNutrition_Patches
+{
+    public static bool IsAttachedToNet(CompPawnStorageNutrition comp, out PipeNet pipeNet, out CompResource resource)
+    {
+        resource = comp.parent.GetComp<CompResource>();
+        pipeNet = null;
+        if (resource is not { PipeNet: { } net }) return false;
+        pipeNet = net;
+        return pipeNet.connectors.Count > 1;
```

Comment:
surely `>=1`? Unless this is counting itself though they could all be consumers regardless no?